tests/backup/backup_helper.go

```diff
@@ -6545,3 +6545,47 @@ func SuspendAndDeleteSchedule(backupScheduleName string, schedulePolicyName stri
 	}
 	return nil
 }
+
+// ValidateCustomResourceRestore validates restore taken of custom resource
+func ValidateCustomResourceRestore(ctx context.Context, orgID string, resourceTypesFilter []string, restoreNamespaceScMappingMap map[string][]map[string]string, restoreContextMap map[string][]*scheduler.Context) error {
+	errChan := make(chan error, 100)
+	var errList []error
+	var wg sync.WaitGroup
+	for restoreName, nsScMapping := range restoreNamespaceScMappingMap {
```

Comment:
Add a sized check for `nsScMapping` before triggering the go routine

Reply:
done

Reply:
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/byoc/job/px-backup-on-demand-system-test-byoc/3910/consoleFull